src/client/routes/PacketDetailsPage/index.js

```diff
@@ -64,12 +70,26 @@ const PacketDetailsPage = () => {
     if (signURL) window.location.assign(signURL)
   }
 
+  const handleOpenSignFrame = async () => {
+    const signURL = await generateSignURL() + '&withinIframe=true'
```

Comment:
Ah, the user shouldnt need to set the withinIframe param, the SignatureFrame component should do it...

Reply:
Right, added this in